evm/src/cpu/kernel/asm/memory/core.asm

```diff
@@ -33,6 +253,13 @@
     // stack: value
 %endmacro
 
+// Load a single byte from user general memory.
+%macro mload_current_memory
+    // stack: offset
+    %mload_current(@SEGMENT_MEMORY)
```

Comment:
What is `SEGMENT_MEMORY`? Do you mean `SEGMENT_MAIN_MEMORY`?

Reply:
Oops, this was added by accident I think